django/db/models/base.py

```diff
@@ -212,17 +215,22 @@ def __new__(cls, name, bases, attrs):
                 continue
 
             parent_fields = base._meta.local_fields + base._meta.local_many_to_many
-            # Check for clashes between locally declared fields and those
-            # on the base classes (we cannot handle shadowed fields at the
-            # moment).
-            for field in parent_fields:
-                if field.name in field_names:
-                    raise FieldError(
-                        'Local field %r in class %r clashes '
-                        'with field of similar name from '
-                        'base class %r' % (field.name, name, base.__name__)
-                    )
             if not base._meta.abstract:
+
+                # Check for clashes between locally declared fields and those
+                # on the base classes (we cannot handle shadowed fields from
+                # concrete models at the moment).
```

Comment:
Regarding the comment "we cannot handle shadowed fields from concrete models at the moment", do you mean one concrete model shadowing fields from another concrete model in a multiple-table inheritance scenario? If that's case we can just remove the comment since we will never support it.